internal/resources/cluster/tkgaws/resource_tkg_aws.go

```diff
@@ -615,19 +619,30 @@ func expandTKGAWSTopology(data []interface{}) (topology *tkgawsmodel.VmwareTanzu
 		return topology
 	}
 
-	lookUpTopology, _ := data[0].(map[string]interface{})
 	topology = &tkgawsmodel.VmwareTanzuManageV1alpha1ClusterInfrastructureTkgawsTopology{}
 
+	lookUpTopology, ok := data[0].(map[string]interface{})
```

Comment:
This ok check is not needed. 

ref : 

`import "fmt"

func main() {

	var data interface{}

	lookUpTopology, _ := data.(map[string]interface{})

	value, ok := lookUpTopology["something"]

	fmt.Println(value, ok)

}
`

the code does not crash